pages/en/web/desktop-progressive-web-apps.md

```diff
@@ -40,7 +40,7 @@ With your PWA installable, it’ll now show up across Chrome OS, from the launch
 
 ### A note on working offline
 
-There are plenty of discoverability, usability, and capability advantages simply by making your app installable. Making your PWA installable does _not_ mean your whole experience must work offline. In order for an installed app to feel native, though, it should have some form of offline functionality. A basic custom offline page is enough to get started; users expect that installed apps don’t crash when connection status changes. Just as a native app never shows a blank page when it is offline, PWAs should never show the browser default offline page. Custom offline pages can range from a message letting the user know they need a connection to a game to let them pass the time until they’re connected again. Providing this custom offline experience for uncached pages or features that require a connection helps to bridge the native-web user experience app gap.
+There are plenty of discoverability, usability, and capability advantages simply by making your app installable. Making your PWA installable does _not_ mean your whole experience must work offline. In order for an installed app to feel like a compiled app, though, it should have some form of offline functionality. A basic custom offline page is enough to get started; users expect that installed apps don’t crash when connection status changes. Just as a compiled app never shows a blank page when it is offline, PWAs should never show the browser default offline page. Custom offline pages can range from a message letting the user know they need a connection to a game to let them pass the time until they’re connected again. Providing this custom offline experience for uncached pages or features that require a connection helps to bridge the compiled-web user experience app gap.
```

Comment:
again, maybe "desktop" instead of compiled for first 2 changes?

Last line gets weird and jargon-y at the end, what about: "Providing a good, customized offline experience offers users the desktop experience they expect from an installed app."